src/app/pages/account/groups/group-form/group-form.component.ts

```diff
@@ -54,7 +127,13 @@ export class GroupFormComponent {
     private cdr: ChangeDetectorRef,
     private errorHandler: FormErrorHandlerService,
     private translate: TranslateService,
-  ) {}
+    public ixUsersProvider: IxUserComboboxProviderService,
+  ) {
+    this.userProvider.providerUpdater$.pipe(untilDestroyed(this)).subscribe(() => {
```

Comment:
I'm struggling to understand the point of this. Is this to make change detection work? If so, then `ix-combobox2` should be responsible for rerunning change detection after new options have been received from the provider.